sw/device/lib/dif/dif_otbn_unittest.cc

```diff
@@ -386,5 +386,26 @@ TEST_F(DmemReadTest, SuccessWithOffset) {
   EXPECT_EQ(test_data[1], 0xabcdef01);
 }
 
+class ControlSoftwareErrorsFatalTest : public OtbnTest {};
+
+TEST_F(ControlSoftwareErrorsFatalTest, NullArgs) {
+  EXPECT_EQ(dif_otbn_set_ctrl_software_errs_fatal(nullptr, false), kDifBadArg);
+}
+
+TEST_F(ControlSoftwareErrorsFatalTest, Success) {
+  EXPECT_WRITE32(OTBN_CTRL_REG_OFFSET, 0x1);
+  EXPECT_READ32(OTBN_CTRL_REG_OFFSET, 0x1);
+
+  EXPECT_EQ(dif_otbn_set_ctrl_software_errs_fatal(&dif_otbn_, true), kDifOk);
+}  // namespace
```

Comment:
nit:
```suggestion
}
```

Reply:
The namespace is closed a couple lines further down, this is only the closing } for the TEST_F section.

Reply:
Ah, I wasn't looking at the line numbers!